api/server/routes/endpoints/anthropic/initializeClient.js

```diff
@@ -2,9 +2,8 @@ const { AnthropicClient } = require('../../../../app');
 const { getUserKey, checkUserKeyExpiry } = require('../../../services/UserService');
 
 const initializeClient = async ({ req, res }) => {
-  const { ANTHROPIC_API_KEY } = process.env;
+  const { ANTHROPIC_API_KEY,ANTHROPIC_REVERSE_PROXY } = process.env;
```

Comment:
make sure you have eslint installed. Please read the contributing guide for best practices here: https://github.com/danny-avila/LibreChat/blob/main/.github/CONTRIBUTING.md